lib/features/receipt/presentation/widgets/list_entry/purchase_receipt_list_entry.dart

```diff
@@ -10,18 +11,32 @@ class PurchaseReceiptListEntry extends StatelessWidget {
     required this.receipt,
   });
 
+  String get priceText {
+    final price = receipt.price;
+    return switch (receipt.paymentStatus) {
+      PaymentStatus.completed => '$price,-',
```

Comment:
perhaps we should include a function formatPrice in strings.dart that handles formatting currency related strings?

Reply:
Might agree, but out of scope of this PR

Reply:
there is already a price function in Strings.dart l. 159. I would prefer if this was fixed now while we are fixing stuff here